casanovo/denovo/model_runner.py

```diff
@@ -255,7 +262,36 @@ def initialize_trainer(self, train: bool) -> None:
                 strategy=self._get_strategy(),
                 val_check_interval=self.config.val_check_interval,
                 check_val_every_n_epoch=None,
+                log_every_n_steps=1,
```

Comment:
**question:** Shouldn't this use the logging interval from the config?

Reply:
Yep, updated.